gslib/commands/cp.py

```diff
@@ -338,9 +338,9 @@
 
   Until the upload has completed successfully, it is not visible at the destination
   object and does not supersede any existing object the upload is intended to
-  replace. However, parallel composite uploads may leave temporary component
-  objects in place during the upload process. See Parallel Composite Uploads for more
-  information.
+  replace. However, `parallel composite uploads
+  <https://cloud.google.com/storage/docs/uploads-downloads#parallel-composite-uploads>`_
```

Comment:
Is the "_" intentional?

Reply:
Yes - though I have it here because I'm copying the style used in existing links, such as the one on line 414 below. As to the purpose (or need) for it, I can only speculate it's somehow required as part of the github --> cloud.google.com generation process.